usage/conf/routes

```diff
@@ -1,5 +1,8 @@
 GET     /                                               controllers.UsageApi.index
 
+GET     /usage/media/:mediaId                           controllers.UsageApi.forMedia(mediaId: String)
+GET     /usage/content/:contentId                       controllers.UsageApi.forContent(contentId: String)
```

Comment:
TO query via a content ID or a media ID. We need to gather more requirements for this.